docker/docker-compose-2.9.7.yml

```diff
@@ -46,7 +46,7 @@ services:
     depends_on: 
       - db-2.9.7
     # command: bash -c "tail -f /dev/null"
-    command: bash -c "python $CKAN_VENV/src/pycsw/pycsw/wsgi.py"
+    command: bash -c "python /usr/lib/ckan/venv/src/pycsw/pycsw/wsgi.py"
```

Comment:
and this

Reply:
docker-compose file for 2.9.7 has been removed